internal/compiler/widgets/material/color-scheme.slint

```diff
@@ -0,0 +1,6 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+export global ColorSchemeSelector := {
+    property<bool> dark-color-scheme: SlintInternal.dark-color-scheme();
```

Comment:
I changed this to be a problem in https://github.com/slint-ui/slint/pull/1790 so it can be simplified with the next rebase:

```suggestion
    property<bool> dark-color-scheme: SlintInternal.dark-color-scheme;
```